docs/contributing/development-policies.rst

```diff
@@ -0,0 +1,219 @@
+until the team has more expert developers in those areas.
+
+
+**********************************************
+Proposal and discussion of significant changes
+**********************************************
+
+New features and backward-incompatible changes should be proposed using the `django CMS developers email list
+<http://groups.google.com/group/django-cms-developers>`_. Discussion should take place there before any pull requests
+are made.
+
+This is in the interests of openness and transparency, and to give the community a chance to participate in and
+understand the decisions taken by the project.
+
+
+****************
+Release schedule
+****************
+
+The `roadmap <https://github.com/divio/django-cms/wiki/Roadmap>`_ can be found on our GitHub wiki
+page.
+
+We are planning releases according to **key principles and aims**. Issues within milestones are
+therefore subject to change.
+
+The `django CMS developers email list <http://groups.google.com/group/django-cms-developers>`_ serves as gathering
+point for developers. We submit ideas and proposals prior to the roadmap goals.
+
+django CMS 3.4 will be the first "LTS" ("Long-Term Support") release of the application. *Long-term support* means that
+this version will continue to receive security and other critical updates for 24 months after its first release.
+
+Any updates it does receive will be backward-compatible and will not alter functional behaviour. This means that users
+can deploy this version confident that keeping it up-to-date requires only easily-applied security and other critical
+updates, until the next LTS release.
+
+
+.. _branch_policy:
+
+********
+Branches
+********
+
+..  versionchanged:: 3.3
+
+    Previously, we maintained a ``master`` branch (now deleted), and a set of ``support`` branches (now pruned, and
+    renamed ``release``).
+
+We maintain a number of branches on `our GitHub repository <https://github.com/divio/django-cms>`_.
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI.
+
+    **Fixes and backwards-compatible improvements** (i.e. most pull requests) will be made against
+    this branch.
+
+``develop``
+    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
+    ``release/x.y.z``.
+
+    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
+    substantial team co-ordination.
+
+Older ``release/x.y.z`` branches
+     These represent the final point of development (the highest ``y`` of older versions). Releases in the full set of
+     older versions have been tagged (use Git Tags to retrieve them).
+
+     These branches will only rarely be patched, with security fixes representing the main reason for a patch.
+
+Commits in ``release/x.y.z`` will be merged forward into ``develop`` periodically by the core developers.
```

Comment:
Pull requests need to be forward ported not just a general "let's move everything now" as discussed. We might want to clarify this here.